.github/workflows/spread-tests.yaml

```diff
@@ -378,6 +400,22 @@ jobs:
             echo "TEST_FAILED=true" >> $GITHUB_ENV
           fi
 
+    - name: Prepare artifact upload
+      if: always() && fromJSON(inputs.upload-artifacts)
+      run: |
+        echo "${{ github.run_attempt }}" > "${ARTIFACTS_FOLDER}/run-attempt.txt"
+        cp "$FAILED_TESTS_FILE" "${ARTIFACTS_FOLDER}/failed-tests.txt"
+        echo "SPREAD_EXPERIMENTAL_FEATURES=${SPREAD_EXPERIMENTAL_FEATURES}" > "${ARTIFACTS_FOLDER}/env-variables.txt"
+        echo "SPREAD_SNAPD_DEB_FROM_REPO=${SPREAD_SNAPD_DEB_FROM_REPO}" >> "${ARTIFACTS_FOLDER}/env-variables.txt"
+        tar -czf "spread-artifacts-${{ inputs.group }}-${{ inputs.systems }}_${{ github.run_attempt }}.tar.gz" "$ARTIFACTS_FOLDER"
```

Comment:
I see this is not being used the ${{ github.run_id }}, it could cause an artifacts overlap, right?